### ISSUE TITLE
Problems with DVR Hikvision DS-7204HGHI-SH / A

### ISSUE THREAD

**Issue by alek4k** [NorthernMan54/homebridge-hikvision-local#2]
### Current Situation

Hi! I'm trying your plugin with my DVR Hikvision DS-7204HGHI-SH / A.
With version 2.0.0, the cameras are detected, the preview image works, and motion events are functioning properly. 
However, when I try to view the live video in HomeKit, I see this error in the HomeBridge console: 
```
[10/10/2024, 8:40:56 PM] [Hikvision-Local] [Camera 01] Starting video stream: 1280 x 720, 30 fps, 299 kbps (AAC-eld)
[10/10/2024, 8:40:58 PM] [Hikvision-Local] [Camera 01] FFmpeg exited with code: 234 and signal: null (Error)
[10/10/2024, 8:40:58 PM] [Hikvision-Local] [Camera 01] Stopped video stream.
```
With version 2.0.1, on the other hand, I immediately get this error when the plugin starts: 
```
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Connected to NVR system: @ http://192.168.178.3 -> 'Embedded Net DVR' DS-7204HGHI-SH/A
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Loading cameras...
TypeError: Cannot read properties of undefined (reading 'model')
    at /homebridge/node_modules/homebridge-hikvision-local/src/HikVisionNVR.ts:56:52
    at Array.map (<anonymous>)
    at HikVisionNVR.<anonymous> (/homebridge/node_modules/homebridge-hikvision-local/src/HikVisionNVR.ts:43:18)
    at Generator.next (<anonymous>)
    at fulfilled (/homebridge/node_modules/homebridge-hikvision-local/dist/HikVisionNVR.js:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Child bridge process ended
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Process Ended. Code: 1, Signal: null
```
With HomeBridge Camera FFmpeg plugin, I can see the camera video streams correctly. 
Do you have any suggestions?
Thank you very much for your work!

### Logs

```markdown
v.2.0.0
[10/10/2024, 8:40:56 PM] [Hikvision-Local] [Camera 01] Starting video stream: 1280 x 720, 30 fps, 299 kbps (AAC-eld)
[10/10/2024, 8:40:58 PM] [Hikvision-Local] [Camera 01] FFmpeg exited with code: 234 and signal: null (Error)
[10/10/2024, 8:40:58 PM] [Hikvision-Local] [Camera 01] Stopped video stream.

Additional error log with debug ffmpeg enabled:
[10/10/2024, 9:01:14 PM] [Hikvision-Local] [Camera 01] [info] Output #1, rtp, to 'srtp://192.168.178.69:55768?rtcpport=55768&pkt_size=188':
[10/10/2024, 9:01:14 PM] [Hikvision-Local] [Camera 01] [out#1/rtp @ 0xffffa1ecc030] [error] Output file does not contain any stream
[10/10/2024, 9:01:14 PM] [Hikvision-Local] [Camera 01] [error] Error opening output file srtp://192.168.178.69:55768?rtcpport=55768&pkt_size=188.
[10/10/2024, 9:01:14 PM] [Hikvision-Local] [Camera 01] [fatal] Error opening output files: Invalid argument
[10/10/2024, 9:01:14 PM] [Hikvision-Local] [Camera 01] [AVIOContext @ 0xffffa1ecd020] [verbose] Statistics: 0 bytes written, 0 seeks, 0 writeouts
[10/10/2024, 9:01:14 PM] [Hikvision-Local] [Camera 01] FFmpeg exited with code: 234 and signal: null (Error)

v.2.0.1
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Connected to NVR system: @ http://192.168.178.3 -> 'Embedded Net DVR' DS-7204HGHI-SH/A
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Loading cameras...
TypeError: Cannot read properties of undefined (reading 'model')
    at /homebridge/node_modules/homebridge-hikvision-local/src/HikVisionNVR.ts:56:52
    at Array.map (<anonymous>)
    at HikVisionNVR.<anonymous> (/homebridge/node_modules/homebridge-hikvision-local/src/HikVisionNVR.ts:43:18)
    at Generator.next (<anonymous>)
    at fulfilled (/homebridge/node_modules/homebridge-hikvision-local/dist/HikVisionNVR.js:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Child bridge process ended
[10/10/2024, 8:46:30 PM] [Hikvision-Local] Process Ended. Code: 1, Signal: null
```

### Configuration

```json
{
    "name": "Hikvision-Local",
    "host": "192.168.178.3",
    "port": 80,
    "secure": false,
    "username": "myUsername",
    "password": "myPassword",
    "debugFfmpeg": false,
    "_bridge": {
        "username": "xx:xx:xx:xx:xx:xx",
        "port": xxxxx
    },
    "platform": "Hikvision-Local"
}
```

### Environment

- OS: Ubuntu Jammy (22.04.5 LTS)
- Software: Homebridge v1.8.4
- Node: v20.18.0
- npm: 10.8.2

### Process Supervisor

Docker (Mention image name in `Additional Context`)

**Comment by NorthernMan54** [comment]
Sorry about the delay on this, but I must have missing the notification.

I believe I have fixed the issue if you can try `v2.0.2-beta.3`.  If the issue does re-occur, I added a debug logging flag in the configuration, that will collected more details about the issue.

Tks

**Comment by alek4k** [comment]
Don't worry :)
I can confirm that with version `v2.0.2-beta.3` it works perfectly! 
Thanks again for your precious work
